.betterer.results

```diff
@@ -3114,6 +3114,12 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
+    "public/app/core/components/PageNew/PageTabs.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"]
+    ],
+    "public/app/core/components/PageNew/SectionNav.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"]
+    ],
```

Comment:
this is fine for now 👍 think we'll need to move the `IconName` interface to `@grafana/data` in order to properly fix this? 🤔